.jshintrc

```diff
@@ -14,12 +14,13 @@
     "Cocktail",
     "_",
     "videojs",
-    "Audio5js",
+    "Howl",
+    "Howler",
```

Comment:
Could not find any usages of `Howler` global. Only `Howl`.

Reply:
It was for the global unlock (Howler._enableMobileAudio()).
But if not necessary, I'll remove.